src/main/webapp/app/exam/manage/student-exams/student-exam-detail.component.ts

```diff
@@ -178,7 +178,7 @@ export class StudentExamDetailComponent implements OnInit {
      */
     updateWorkingTimeDuration() {
         const regularWorkingTime = normalWorkingTime(this.studentExam.exam!)!;
-        const seconds = Math.round(regularWorkingTime * (1.0 + this.workingTimeFormValues.percent / 100));
+        const seconds = round(regularWorkingTime * (1.0 + this.workingTimeFormValues.percent / 100), 0);
```

Comment:
Do you think it makes sense to place this in the exam utils as well? So you have there both directions. Even though it doesn't get used in other places so far but it probably might be better maintanable.

Reply:
I don’t think so. This component is and should remain the only place where an instructor can edit the working time. All other places only ever know about the seconds and calculate the extension from there, never the other way round.